README.md

```diff
@@ -125,11 +129,15 @@ compression_ctrl, compressed_model = create_compressed_model(model, nncf_config)
 
 # ... the rest of the usual TensorFlow-powered training pipeline
 
-# Export to Frozen Graph, TensorFlow SavedModel or .h5  when done fine-tuning 
+# (Optional) Preparing the model to work in framework native format without NNCF specific operations to inference
+# or export the model.
+inference_model = compression_ctrl.prepare_for_inference(make_model_copy=True)
```

Comment:
IMHO. The sample script should not include optional way.

BTW, Do you have behavior tests of using prepare_for_inference() and export_model() together?

Reply:
No, should used one of this methods.
But by defaults `prepare_for_inference` is not changes original model in controller, so`export_model` works without any changes.